### PR TITLE
feat: add notebook to tutorial

### PR DESCRIPTION
**Pull request by jpetot**
- feat: add notebooks to tutorial
- add like to tutorial

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@ DOC_OUTPUT_DIR ?= doc/build/html# will read from DOC_OUTPUT_DIR environment vari
 DOC_SOURCE_DIR := doc/source
 
 doc: doc-build  ## Build and open the docs
-	python -m webbrowser -t $(DOC_OUTPUT_DIR)/index.html
+	poetry run python -m webbrowser -t $(DOC_OUTPUT_DIR)/index.html
```
Comment (charlax) [inline Makefile:20]:
```suggestion
	python -m webbrowser -t $(DOC_OUTPUT_DIR)/index.html
```

`webbrowser` is in the standardlib, no need for poetry.

**Review by olivierabz — APPROVED** [review]
Small suggestions on the nb:
- Make the title match the name of the section: "Evaluating avatar utility"
- Add headings in the nb (ideally matching the 3 criteria you mention in `tutorial.md` (univariate, bivariate, multivariate)

**Review by mguillaudeux — COMMENTED** [review]
Useful to have !
As evaluate_quality.ipynb aims to be a tutorial, I would give more insight to the user about the interest of the different analyses.
We could for instance:
- Clearly separate univariate, bivariate and multivariate sections 
- Explain what is the purpose of comparing distributions, correlations and projections

**Review by mguillaudeux — APPROVED** [review]
Nice ! We can even send a link the Preparing section of our documentation prior any comparison

**Pull request merged**